samples/csharp_dotnetcore/48.qnamaker-active-learning-bot/Dialog/DialogHelper.cs

```diff
@@ -75,7 +75,7 @@ private async Task<DialogTurnResult> CallGenerateAnswer(WaterfallStepContext ste
 
             stepContext.Values[QnAData] = new List<QueryResult>(filteredResponse);
             stepContext.Values[CurrentQuery] = stepContext.Context.Activity.Text;
-            return await stepContext.NextAsync(cancellationToken);
+            return await stepContext.NextAsync(stepContext.Context, cancellationToken);
```

Comment:
OK definitely a bug to pass the cancellationToken in as the result. However, this is now passing the current TurnContext in as the result. That doesn't seem correct either.

The idea of the result is that it is the result of this step that arrives as the input value of the next step.